src/premiumvideodevice/PremiumVideoEffectDriver.ts

```diff
@@ -0,0 +1,67 @@
+import Logger from '../logger/Logger';
+import PremiumVideoEffectConfig from './PremiumVideoEffectConfig';
+
+const CHANNEL_COUNT = 4; // The number of channels in a video frame
```

Comment:
maybe consider moving all the constant in a separate file. It has pro and cons though. I would like to hear Faraz decision here!